src/GroupParametersInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Router;
+
+interface GroupParametersInterface
```

Comment:
Why? Has use case for read groups in application?

Reply:
Deleted.